tests/test_wave_03.py

```diff
@@ -54,32 +54,22 @@ def test_friends_unique_movies_not_duplicated():
 
     # Assert
     assert len(friends_unique_movies) == 3
+    assert len(set(movie["title"] for movie in friends_unique_movies)) == 3
```

Comment:
I'm taking this assert as your attempt to ensure the duplicated movie was not repeated, which is really what we should be testing here instead of our recommendation to check for the correct movies.

However, this assert alone wouldn't find such a duplication, as by deduping using `set` would obviously hide any duplicates. I do think that combined with the assert on line 56 it does verify that though, as if the number of items in a list and a deduped version of that list are the same, then the original list must not have duplicates.

I would probably combine the two asserts like `assert len(friends_unique_movies) == len(set(movie["title"] for movie in friends_unique_movies))`, but your code is fine as is, I guess I wanted to hear myself talk.

That being said, there could also be a concern that there are multiple movies with the exact same `"title"` key (perhaps the 1960 and 1998 versions of Alfred Hitchcock's Psycho?). However none of that shows up in our test data, and it's possible they would distinguish these movies like "Psycho" and "Psycho (1998)".